### PR TITLE
Brewfile and docs

### PR DESCRIPTION
**Pull request by levkk**

### CODE REVIEW

File: pgml-dashboard/content/.ignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*
```
Comment (montanalow) [inline pgml-dashboard/content/.ignore:1]: Why are you ignoring everything here?

File: pgml-dashboard/content/docs/guides/setup/v2/installation.md [hunk 2]
```diff
@@ -1,318 +1,298 @@
 # Installation
 
-The PostgresML deployment consists of two parts: the Postgres extension and the dashboard app. The extension provides all the machine learning functionality and can be used independently. The dashboard app provides a system overview for easier management and notebooks for writing experiments.
+The PostgresML deployment consists of two parts: the PostgreSLQ extension, and the dashboard app. The extension provides all the machine learning functionality and can be used independently. The dashboard app provides a system overview for easier management and notebooks for writing experiments.
```
Comment (montanalow) [inline pgml-dashboard/content/docs/guides/setup/v2/installation.md:3]:
```suggestion
The PostgresML deployment consists of two parts: the PostgreSQL extension, and the dashboard app. The extension provides all the machine learning functionality and can be used independently. The dashboard app provides a system overview for easier management and notebooks for writing experiments.
```

File: pgml-dashboard/content/.ignore (hunk 1 above)
Comment (levkk) [reply pgml-dashboard/content/.ignore:1]: This is the `.ignore` for cargo-watch, not for git.

**Comment by levkk** [comment]
@montanalow Finally fixed the duplicate heading ids in our toc, atomic counters ftw!

**Pull request merged**